core/src/components/datetime/test/set-value/datetime.e2e.ts

```diff
@@ -49,11 +50,9 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       await page.waitForChanges();
 
       // Select October 2021
+      // The year will automatically switch to 2021 when selecting 10
       await monthColumn.locator('.picker-item[data-value="10"]').click();
-      await page.waitForChanges();
-
-      await yearColumn.locator('.picker-item[data-value="2021"]').click();
-      await page.waitForChanges();
+      await ionChange.next();
```

Comment:
This test also started to fail only in Firefox. The problem was that tapping the "October" button  started to scroll, but we closed the month/year picker before the scroll finished so the value was never updated. By waiting for `ionChange` we can ensure the value is updated.